packages/core/admin/ee/server/tests/review-workflows.test.api.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const { createStrapiInstance } = require('../../../../../../test/helpers/strapi');
+const { createAuthRequest, createRequest } = require('../../../../../../test/helpers/request');
+
+const edition = process.env.STRAPI_DISABLE_EE === 'true' ? 'CE' : 'EE';
+
+if (edition === 'CE') {
+  test('Review-workflows (skipped)', () => {
+    expect(true).toBeTruthy();
+  });
+  return;
+}
+
+describe('Review workflows', () => {
+  const requests = {
+    public: null,
+    admin: null,
+  };
+  let strapi;
+  let hasRW;
+  let defaultWorkflow;
+
+  beforeAll(async () => {
+    strapi = await createStrapiInstance();
+    // eslint-disable-next-line node/no-extraneous-require
+    hasRW = require('@strapi/strapi/lib/utils/ee').features.isEnabled('review-workflows');
+
+    requests.public = createRequest({ strapi });
+    requests.admin = await createAuthRequest({ strapi });
+
+    defaultWorkflow = await strapi.query('admin::workflow').create({ data: {} });
+  });
+
+  afterAll(async () => {
+    await strapi.destroy();
+  });
+
+  describe('Get workflows', () => {
+    test('It should be available for every connected users (public)', async () => {
```

Comment:
I think you might have typos in these public user test cases? e.g. should it be 'It should not be available for public users'

Reply:
Sure, forgot to replace the text 😅
Nice catch